plutus-tx/src/PlutusTx/Builtins.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE FlexibleContexts #-}
 -- editorconfig-checker-disable-file
```

Comment:
Nit: move `LANGUAGE` to the other language pragmas? Or at least below the editorconfig thing.